s.el

```diff
@@ -271,7 +271,7 @@ attention to case differences."
   "Is S1 equal to S2?
 
 This is a simple wrapper around the built-in `string-equal'."
```

Comment:
I suggest either updating the docstring or moving the normalised comparison to a new function.